src/common/vitals/first-input-delay.js

```diff
@@ -1,19 +1,24 @@
-import { onFID } from 'web-vitals'
+import { onFID } from 'web-vitals/attribution'
 import { VitalMetric } from './vital-metric'
 import { VITAL_NAMES } from './constants'
 import { initiallyHidden, isBrowserScope } from '../constants/runtime'
 
 export const firstInputDelay = new VitalMetric(VITAL_NAMES.FIRST_INPUT_DELAY)
 
 if (isBrowserScope) {
-  onFID(({ value, entries }) => {
-    if (initiallyHidden || firstInputDelay.isValid || entries.length === 0) return
+  onFID(({ value, attribution }) => {
+    if (initiallyHidden || firstInputDelay.isValid) return
+    const attrs = {
+      type: attribution.eventType,
+      fid: Math.round(value),
+      eventTarget: attribution.eventTarget,
+      loadState: attribution.loadState
+    }
 
     // CWV will only report one (THE) first-input entry to us; fid isn't reported if there are no user interactions occurs before the *first* page hiding.
     firstInputDelay.update({
-      value: entries[0].startTime,
-      entries,
```

Comment:
you're talking about `entries` right? Yeah it's never used except by the current `onTTFB` for the load node (and technically if we moved onTTFB to attribution, it wouldn't need `entries` either as `attribution.navigationEntry` would hold it).

Entries is a bit useless atm and just extra references to perf entries.

Reply:
I put this comment on several changes. I just want to make sure that we are either not using the `entries` property or accounted for the change. Also, just want to make sure that the final harvest payload is accurate and not missing something because of this change. Just precautionary here.

Reply:
Good to double check. So only `onTTFB` is subscribed in more than the one PVT aggregate code. TTFB was not switched over here aka still uses the non-attribution build, and the `entries` for it was left alone as well as in the VitalMetric class, so yeah nothing's missing or breaking with this.
(onTTFB could be switched and this `entries` removed altogether really, but unnecessary work.)

Reply:
As long as we have a predictable way to access data regardless of the metric type, i'm good with it.  I don;t love that most dont use entries, but one does.  if it feels disjointed as is we may need to revisit standardizing the attributes flowing through these emitters at a later time

Reply:
Ported ttfb over and removed `entries` entirely at this time as it's not meaningfully used anywhere.